browser/mocha/stub.js

```diff
@@ -25,20 +33,17 @@ extendInterfaces('stub', function(context, teardown) {
     var proto = document.createElement(tagName).constructor.prototype;
 
     // For all keys in the implementation to stub with..
-    var keys = Object.keys(implementation);
-    keys.forEach(function(key) {
+    var stubs = Object.keys(implementation).map(function(key) {
```

Comment:
Lets keep this code as-is and not make a change here unnecessarily. Renaming keys to stubs is 👍 , but changing `forEach()` to `map()` and using the `sinon.stub()` return value now hides the fact that sinon monkey patches objects in place.